code/modules/mob/living/emote.dm

```diff
@@ -244,6 +244,15 @@
 	message_mime = "acts out a scream!"
 	emote_type = EMOTE_AUDIBLE
 
+/datum/emote/living/scream/run_emote(mob/user, params)
+	. = ..()
+	if(!.)
+		return ..()
+	if(ishuman(user))
+		var/mob/living/carbon/human/H = user
+		if(H.dna.species.scream_sound)
+			playsound(H.loc, H.dna.species.scream_sound, 50, 1, 4, 1.2)

```

Comment:
I do not think this is a good idea

Reply:
>no cooldown


GOD NO.

Reply:
nonsense this is fantastic

Reply:
It's a port from hippie's screamsound mechanics. This is an unlogged change that should probably get added to the changelog.

Reply:
It also calls `..()` twice in some situations which may be a bad